src/SettingsTab.ts

```diff
@@ -64,5 +64,25 @@ export class SettingsTab extends PluginSettingTab {
                         await this.plugin.saveSettings();
                     });
             });
+
+        containerEl.createEl('div', {
+            cls: 'setting-item-description',
+            text: 'Enable reminder on edit/create task (requires obsidian-reminder plugin)',
```

Comment:
Does it actually require the reminder plugin, or is it there is only really any benefit of you are using the reminders plugin?

(By which I mean, do we need to add code to check if the reminders plugin is enabled?)

Is `obsidian-reminder` the plugin id? Would it be better to use the plugin name that users will see in the Plugin store?